### PR TITLE
feat: export pluralize functionality

### PR DESCRIPTION
**Pull request by imurchie**
Add export of https://github.com/blakeembrey/pluralize through the `util` module.

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -338,10 +339,21 @@ function unleakString (s) {
   return ` ${s}`.substr(1);
 }
 
+/**
+ * Get the form of a word appropriate to the count
+ *
+ * @param {string} word - The word to pluralize
+ * @param {number} count - How many of the word exist
+ * @param {?boolean} inclusive - Whether to prefix with the number (e.g., 3 ducks)
```
Comment (mykola-mokhnach) [inline lib/util.js]: `inclusive [false]`
I'd also think about putting all options into a dictionary, so we could extend it later if needed

Comment (imurchie) [reply lib/util.js]: Sure.

File: lib/util.js [hunk 2]
```diff
@@ -338,10 +339,33 @@ function unleakString (s) {
...
+ * @property {?boolean} inclusive [false] - Whether to prefix with the number (e.g., 3 ducks)
+ */
+
+/**
+ * Get the form of a word appropriate to the count
+ *
+ * @param {PluralizeOptions} options - options for word pluralization
+ * @returns {string} The word pluralized according to the number
+ */
+function pluralize (options = {}) {
```
Comment (mykola-mokhnach) [inline lib/util.js]: the word and the count may stay as ordinary args, since they are always required

File: lib/util.js [hunk 3]
```diff
@@ -342,23 +342,27 @@ function unleakString (s) {
...
  * @returns {string} The word pluralized according to the number
  */
-function pluralize (options = {}) {
-  const {
-    word,
-    count,
-    inclusive = false,
-  } = options;
+function pluralize (word, count, options = {}) {
+  let inclusive = false;
```
Comment (mykola-mokhnach) [inline lib/util.js]: 👍

**Pull request merged**